src/snowflake/snowpark/window.py

```diff
@@ -246,25 +246,6 @@ def range_between(self, start: int, end: int) -> "WindowSpec":
             SpecifiedWindowFrame(RangeFrame(), boundary_start, boundary_end),
         )
 
-    def _convert_boundary_to_expr(
```

Comment:
This is a duplicate function which is never used.
https://github.com/snowflakedb/snowpark-python/blob/main/src/snowflake/snowpark/window.py#L28